ctrlf.el

```diff
@@ -1002,6 +1033,48 @@ search, change back to regexp search."
   (interactive)
   (ctrlf-backward 'regexp))
 
+;;;###autoload
+(defun ctrlf-forward-symbol-at-point ()
+  "Search forward for symbol at point."
+  (interactive)
+  (let ((arg nil)
+        (pos nil))
```

Comment:
Is there a pattern elisp pattern to follow for this? I need to get the values of `arg` and `pos` as they are within the `with-current-buffer` scope, but  I need to use them outside of it.